libs/model/src/contest/FarcasterContestBotMentioned.command.ts

```diff
@@ -0,0 +1,53 @@
+import { logger, type Command } from '@hicommonwealth/core';
+import { config } from '@hicommonwealth/model';
+import * as schemas from '@hicommonwealth/schemas';
+import { NeynarAPIClient } from '@neynar/nodejs-sdk';
+import { models } from '../database';
+import { mustExist } from '../middleware/guards';
+import { emitEvent } from '../utils';
+
+const log = logger(import.meta);
+
+// This webhook processes the "cast.created" event
+// triggered when the contest bot is mentioned on farcaster
+export function FarcasterContestBotMentionedWebhook(): Command<
```

Comment:
Commands should be using imperative names describing the action, for consistency. In this case we are relaying Neynar events to the Outbox... so, we should probably start a convention here like `RelayFarcasterContestBotMentioned` or something similar that's easy to find when looking for all relays